File: Programmers/Lv0/캐릭터의좌표.py
def solution(keyinput, board):
    answer = [0, 0]
    bound = [(board[0] // 2), board[1] // 2]
    for str in keyinput:
        if str == "up" and answer[1] < bound[1]:
            answer[1] += 1
        elif str == "down" and answer[1] > bound[1] * -1:
            answer[1] -= 1
        elif str == "left" and answer[0] > bound[0] * -1:
            answer[0] -= 1
        elif str == "right" and answer[0] < bound[0]:
            answer[0] += 1
    return answer

File: Programmers/Lv0/캐릭터의좌표.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> keyinput, vector<int> board) {
    vector<int> answer(2, 0);
    for (string str : keyinput) {
        if (str == "up" and answer[1] < board[1] / 2)
            answer[1] += 1;
        else if (str == "down" and answer[1] > board[1] / 2 * -1)
            answer[1] -= 1;
        else if (str == "left" and answer[0] > board[0] / 2 * -1)
            answer[0] -= 1;
        else if (str == "right" and answer[0] < board[0] / 2)
            answer[0] += 1;
    }
    return answer;
}